scripts/assets/accessibility_label_exemptions.textproto

```diff
@@ -0,0 +1,27 @@
+exempted_activity: "app/home/HomeActivity"
+exempted_activity: "app/home/recentlyplayed/RecentlyPlayedActivity"
+exempted_activity: "app/mydownloads/MyDownloadsActivity"
+exempted_activity: "app/player/state/testing/StateFragmentTestActivity"
+exempted_activity: "app/profileprogress/ProfilePictureActivity"
+exempted_activity: "app/settings/profile/ProfileEditActivity"
+exempted_activity: "app/splash/SplashActivity"
+exempted_activity: "app/testing/AudioFragmentTestActivity"
+exempted_activity: "app/testing/BindableAdapterTestActivity"
+exempted_activity: "app/testing/ConceptCardFragmentTestActivity"
+exempted_activity: "app/testing/DragDropTestActivity"
+exempted_activity: "app/testing/ExplorationInjectionActivity"
+exempted_activity: "app/testing/ExplorationTestActivity"
+exempted_activity: "app/testing/HomeFragmentTestActivity"
+exempted_activity: "app/testing/HomeTestActivity"
+exempted_activity: "app/testing/HtmlParserTestActivity"
+exempted_activity: "app/testing/ImageRegionSelectionTestActivity"
+exempted_activity: "app/testing/InputInteractionViewTestActivity"
+exempted_activity: "app/testing/LessonThumbnailImageViewTestActivity"
+exempted_activity: "app/testing/NavigationDrawerTestActivity"
+exempted_activity: "app/testing/ProfileChooserFragmentTestActivity"
+exempted_activity: "app/testing/SplashTestActivity"
+exempted_activity: "app/testing/TestFontScaleConfigurationUtilActivity"
+exempted_activity: "app/testing/TopicRevisionTestActivity"
+exempted_activity: "app/testing/TopicTestActivity"
+exempted_activity: "app/testing/TopicTestActivityForStory"
+exempted_activity: "app/walkthrough/WalkthroughActivity"
```

Comment:
@rt4914 do we already have issues tracking these?

Reply:
Not for all.
For example:
`HomeActivity` does not need a label as it works correctly even without it for A11y users.
`NavigationDrawerTestActivity ` we don't need labels for them.


So what are your thoughts on both of these? Should we still add labels for all the files?

Reply:
I think we can ignore the test activities--they're reasonable exemptions (eventually we'll probably delete them by utilizing a generic replacement, anyway).

For the others, shouldn't we always have an accessibility label to be explicit? If so, then we ought to have a TODO for each of the non-test activities to ensure we can remove them from this exemption list & only leave test activities.

WDYT?

Reply:
Yes your suggestion makes sense.

Filed an issue: https://github.com/oppia/oppia-android/issues/3602

Reply:
Thanks @rt4914!